lms/djangoapps/discussion/templates/discussion/maintenance.html

```diff
@@ -0,0 +1,25 @@
+## mako
+
+<%! main_css = "style-discussion-main" %>
+
+<%! from django.utils.translation import ugettext as _ %>
+<%page expression_filter="h"/>
+<%inherit file="../main.html" />
+
+<%block name="bodyclass">discussion</%block>
+
+<%block name="headextra">
+<%include file="../discussion/_js_head_dependencies.html" />
+</%block>
+
+<%block name="content">
+    <div class="alert alert-error" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
```

Comment:
We should use a heading here that indicates the content (i.e., "Forums unavailable"). It can use the `.sr` class.